README.md

```diff
@@ -45,279 +46,412 @@ Sample demo setp by step javascript example are avalible at [Demo Folder][demo-l
 -   OPERATOR_ID=0.0.xxxx
```

Comment:
Hi @vijayshiyani ,
its typo in line 44 `setp`
Thanks

Reply:
fixed